tools/wptrunner/wptrunner/browsers/firefox_android.py

```diff
@@ -197,6 +197,9 @@ def _set_required_prefs(self, profile):
 
         if self.test_type == "wdspec":
             profile.set_preferences({"remote.prefs.recommended": True})
+            profile.set_preferences({
+                "geo.provider.network.url": "https://web-platform.test:8444/webdriver/tests/support/http_handlers/geolocation_override.py"
+            })
```

Comment:
Please note that this will cause a merge conflict with the not yet merged https://github.com/web-platform-tests/wpt/pull/53468